plugins/inputs/sysstat/README.md

```diff
@@ -0,0 +1,441 @@
+# sysstat Input Plugin
+
+Collect [sysstat](https://github.com/sysstat/sysstat) metrics - requires the sysstat
+package installed.
+
+This plugin collects system metrics with the sysstat collector utility `sadc` and parses
+the created binary data file with the `sadf` utility.
+
+### Configuration:
+
+```toml
+# Sysstat metrics collector
+[[inputs.sysstat]]
+  ## Path to the sadc command.
+  sadc_path = "/usr/lib/sa/sadc" # required
+  #
+  #
+  ## Path to the sadf command, if it is not in PATH
+  # sadf_path = "/usr/bin/sadf"
+  #
+  #
+  ## Activities is a list of activities, that are passed as argument to the
+  ## sadc collector utility (e.g: DISK, SNMP etc...)
+  ## The more activities that are added, the more data is collected.
+  # activities = ["DISK"]
```

Comment:
`sadc_path` is commented because I did not define a default value in the code. I can set the Ubuntu default value if you want, then I can comment out the value.